addon/components/md-text.js

```diff
@@ -61,6 +61,19 @@ export default Ember.Component.extend({
       ]);
     }
 
+    let plugins = this.get('plugins');
+    if (plugins) {
+      plugins.forEach((plugin) => {
```

Comment:
Can we try [`md.use`](https://github.com/jonschlinkert/remarkable#plugins) here?